docs/resources/vkcs_networking_secgroup_rule.md

```diff
@@ -59,9 +59,9 @@ resource "vkcs_networking_secgroup_rule" "secgroup_rule_1" {
 
 - `region` **String** (*Optional*) The region in which to obtain the networking client. A networking client is needed to create a port. If omitted, the `region` argument of the provider is used. Changing this creates a new security group rule.
 
-- `remote_group_id` **String** (*Optional*) The remote group id, the value needs to be an ID of a security group in the same tenant. Changing this creates a new security group rule.
+- `remote_group_id` **String** (*Optional*) The remote group id, the value needs to be an ID of a security group in the same tenant. Only one of `remote_group_id` or `remote_ip_prefix` may be set. Changing this creates a new security group rule.
 
-- `remote_ip_prefix` **String** (*Optional*) The remote CIDR, the value needs to be a valid CIDR (i.e. 192.168.0.0/16). Changing this creates a new security group rule.
+- `remote_ip_prefix` **String** (*Optional*) The remote CIDR, the value needs to be a valid CIDR (i.e. 192.168.0.0/16). Only one of `remote_group_id` or `remote_ip_prefix` may be set. Changing this creates a new security group rule.
```

Comment:
Lets place it at the end of the string and prepend it with bolded Note

Reply:
Fixed